web/src/pages/Route/components/DataLoader/Import.tsx

```diff
@@ -0,0 +1,262 @@
+      return <OpenAPI3 />;
+  }
+};
+
+const DataLoaderImport: React.FC<Props> = (props) => {
+  const [form] = Form.useForm();
+  const { formatMessage } = useIntl();
+  const { onClose } = props;
+  const [importType, setImportType] = useState<ImportType>('openapi3');
+  const [uploadFileList, setUploadFileList] = useState<RcFile[]>([]);
+  const [state, setState] = useState<ImportState>('import');
+  const [importResult, setImportResult] = useState<ImportResult>({
+    success: true,
+    data: {},
+  });
+
+  const onFinish = (values: Record<string, string>) => {
+    const formData = new FormData();
+    if (!uploadFileList[0]) {
+      notification.warn({
+        message: formatMessage({ id: 'page.route.button.selectFile' }),
+      });
+      return;
+    }
+    Object.keys(values).forEach((key) => {
+      formData.append(key, values[key]);
+    });
+    formData.append('file', uploadFileList[0]);
+
+    importRoutes(formData).then((r) => {
+      let errorNumber = 0;
+      entityNames.forEach((v) => {
+        errorNumber += r.data[v].failed;
+      });
+
+      setImportResult({
+        success: errorNumber <= 0,
+        data: r.data,
+      });
+      setState('result');
+    });
+  };
+
+  return (
+    <>
+      <Drawer
+        title={formatMessage({ id: 'page.route.data_loader.import_panel' })}
+        width={480}
+        visible={true}
+        onClose={onClose}
+        footer={
+          <div
+            style={{
+              display: state === 'result' ? 'none' : 'flex',
+              justifyContent: 'space-between',
+            }}
+          >
+            <Button onClick={onClose}>{formatMessage({ id: 'component.global.cancel' })}</Button>
+            <Space>
+              <Button
+                type="primary"
+                onClick={() => {
+                  form.submit();
+                }}
+              >
+                {formatMessage({ id: 'component.global.submit' })}
+              </Button>
+            </Space>
+          </div>
+        }
+      >
+        {state === 'import' && (
+          <Form layout="vertical" form={form} onFinish={onFinish} hideRequiredMark>
+            <Row gutter={16}>
+              <Col span={12}>
+                <Form.Item
+                  name="type"
+                  label={formatMessage({ id: 'page.route.data_loader.labels.loader_type' })}
+                  rules={[
+                    {
+                      required: true,
+                      message: formatMessage({ id: 'page.route.data_loader.tips.select_type' }),
+                    },
+                  ]}
+                  initialValue={importType}
+                >
+                  <Select onChange={(value: ImportType) => setImportType(value)}>
+                    <Select.Option value="openapi3">
+                      {formatMessage({ id: 'page.route.data_loader.types.openapi3' })}
+                    </Select.Option>
+                    <Select.Option value="openapi_legacy" disabled>
```

Comment:
The older version is provided by PR #2468, but it is still not merged, so it is disabled here. It is expected that it will be provided, but we will not provide any maintenance for it anymore. It may be removed in the future (after we have a new version of the export implementation), I can't be sure which will happen first.